spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'Changing authentication factor' do
   describe 'requires re-authenticating' do
     before do
-      sign_up_and_2fa
+      @user = sign_up_and_2fa
```

Comment:
Maybe we could table this here and discuss further separately? 18F/identity-private#968